tests/search/components/TestSearchForm.js

```diff
@@ -7,24 +7,30 @@ const render = ReactTestUtils.renderIntoDocument;
 const findByTag = ReactTestUtils.findRenderedDOMComponentWithTag;
 
 describe('<SearchForm />', () => {
+  var onSearch;
```

Comment:
Yep fair enough - I'll move us to let + const. I might do it separate to this patch though since it'll be easier to review a patch that updates everything in one hit.